xdsl/ir.py

```diff
@@ -683,11 +683,6 @@ def verify(self, verify_nested_ops: bool = True) -> None:
             if isinstance(operand, ErasedSSAValue):
                 raise Exception("Erased SSA value is used by the operation")
 
-        # Verifier generated by irdl_op_def
-        op_def = type(self).irdl_definition
-        if op_def is not None:
-            op_def.verify(self)
```

Comment:
So you're not calling IRDL's verify anymore now, right?
I think you could just call `verify_` here for now?